spec/identity_doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -54,7 +54,7 @@
     end
     it 'has PII data' do
       pii_from_doc = described_class.new(success_response, false, config).pii_from_doc
-      expect(pii_from_doc).not_to be_empty
+      expect(pii_from_doc).to include(:first_name, :last_name, :dob, :state)
```

Comment:
can we check the values here? Especially useful for checking the DOB formatting code

Reply:
Done.